system-addon/test/unit/lib/Store.test.js

```diff
@@ -0,0 +1,83 @@
+const {Store} = require("lib/Store.jsm")
+const {actionTypes: at} = require("common/Actions.jsm");
+const {createStore} = require("redux");
+const {addNumberReducer} = require("test/unit/utils");
+
+describe("Store", () => {
+  let sandbox;
+  let store;
+  before(() => {
+    sandbox = sinon.sandbox.create();
+  });
+  beforeEach(() => {
+    store = new Store();
+  });
+  afterEach(() => sandbox.restore());
+  it("should have an empty feeds Set", () => {
+    assert.instanceOf(store.feeds, Set);
+    assert.equal(store.feeds.size, 0, ".feeds.size");
+  });
+  it("should have a redux store at ._store", () => {
+    assert.ok(store._store);
```

Comment:
This is really just asserting that store is non-falsy.  Maybe assert.instanceOf is would be a better fit?  Otherwise, I'd suggest changing the description to more accurately describe what's being tested.